docs/release.md

```diff
@@ -39,29 +39,59 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This is the same release cycle as Kubernetes, but KubeVirt will trail Kubernetes releases
+by about a week:
```

Comment:
This example effectively means that we are promising compatibility of 0.55 with 1.24 at the time of release, while the release was only one week earlier. This would imply that we had already tested with the kubernetes preview and rc version of 1.24 earlier, no?

Reply:
We might add a row where we are explicitly stating the compatibility promise wrt k8s versions we are making. WDYT?

Reply:
> stating the compatibility promise wrt k8s versions we are making
>
Sure, I'll add a section.  Do you know if there are any other places that have a compatibility promise?  I thought I read somewhere kubevirt supports (<k8s versions> - 3).

Reply:
> I just wanted to mention that we are currently creating the 1.24 provider after the official release, thus we would need to change how we are creating k8s providers inside kubevirtci.
>
How long does it usually take to create the provider?  Can we use a k8s release candidate before the official release?

Reply:
> > stating the compatibility promise wrt k8s versions we are making
> >
> Sure, I'll add a section.  Do you know if there are any other places that have a compatibility promise?  I thought I read somewhere kubevirt supports (<k8s versions> - 3).

This is the doc for k8s compatibility: https://github.com/kubevirt/kubevirt/blob/main/docs/kubernetes-compatibility.md#kubevirt-main

Especially the second paragraph is important in this context.

If we didn't want to change anything this would imply that we would release a couple of weeks after the k8s release, whenever the new provider is ready and we have fixed all issues regarding the new provider.

Reply:
I updated the document to include a week for the provider availability.

The compatibility promise will stay the same.

Reply:
ATM we are exactly in that situation where no one seems to have bandwidth to work on the new 1.25 provider. The release has already been a week ago and we still don't have it ready, besides that we need the provider to land in kubevirt and the lanes to get created.

In general I'd say we would have it within a couple of weeks from my gut, but I don't have any number TBH. Maybe take 4 weeks as an upper bound? Not sure. What do others think?

@rmohr @fabiand @awels ?

Reply:
Yes, 4 weeks sounds reasonable. Let's start conservative. Maybe, if we find enough interested people and get the bandwidth we can tighten this later by starting using pre-releases and so forth ...